VocalRaidAssistant.lua

```diff
@@ -53,6 +56,20 @@ function VRA:OnInitialize()
 	self.db.RegisterCallback(self, "OnProfileCopied", "ChangeProfile")
 	self.db.RegisterCallback(self, "OnProfileReset", "ChangeProfile")
 	profile = self.db.profile
+	
+	--Minimap Icon and Broker
+	MyLDB = LibStub:GetLibrary("LibDataBroker-1.1"):NewDataObject(addonName, {
+	type = "launcher",
+	icon = "Interface\\COMMON\\VoiceChat-Speaker",
+	OnClick = function(clickedframe, button)
+		self:ChatCommand()
+	end,
+	OnTooltipShow = function(tooltip)
+          tooltip:SetText(VRA.L["VRANAME"])
+          tooltip:Show()
+     end,
+	})
+	VRA.ICON:Register("MyLDB", MyLDB, self.db.profile.minimapIconTable)
```

Comment:
Somehow button position resets after each login, are we saving to db?

Reply:
Fixed saving to db, also added toggle button in options.